usagov/vote-gov-nvrf-app
# VOTE-2363 Replace with json api path

[VOTE-2363](https://cm-jira.usa.gov/browse/VOTE-2363)

Replace the json paths with the asset path to be used while embedded as a module.
When developing and testing this standalone app, the local files will be used.

Testing steps:
1. Open the local site and confirm all data loads correctly for English. You will need to go through each step.
2. To test the Spanish path, JSONHelper.jsx, edit line 5 to hardcode the spanish code
const locale = "es";
3. On the local site, confirm all data loads correctly for Spanish (the content itself will be in English). You will need to go through each step.
